scripts/quests/bastok/Teak_Me_to_the_Stars.lua

```diff
@@ -0,0 +1,70 @@
+-----------------------------------
+-- Teak Me to the Stars
+-----------------------------------
+-- Log ID: 1, Quest ID: 79
+-- Raibaht : !gotoid 17748012
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.BASTOK, xi.quest.id.bastok.TEAK_ME_TO_THE_STARS)
+
+quest.reward =
+{
+    fame     = 30,
+    fameArea = xi.quest.fame_area.BASTOK,
+    gil      = 2100,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getFameLevel(xi.quest.fame_area.BASTOK) >= 3
+        end,
+
+        [xi.zone.METALWORKS] =
+        {
+            ['Raibaht'] = quest:progressEvent(864, 0, xi.item.GARHADA_TEAK_LUMBER, 2964, 721420799, 40186682, 2, 4095, 262246),
```

Comment:
`['Raibaht'] = quest:progressEvent(864, 0, xi.item.GARHADA_TEAK_LUMBER),`

Reply:
Updated with mission check for additional dialog triggered by option 5 (2 if completed wymrking, 0 otherwise)